en/docs/deploy/get-started/get-started-with-the-management-console.md

```diff
@@ -26,7 +27,7 @@ management console.
 
 The following screen depicts the home screen of the management console.
 
-![server-home-management-console](../../assets/img/deploy/server_home_management_console.png)
+![server-home-management-console](../../../assets/img/deploy/server_home_management_console.png)
```

Comment:
better to have a meaningful text as the alt text for the image (for SEO purposes)